help.txt

```diff
@@ -67,78 +68,110 @@ FUNCTIONS
             Mimicks built-in open parameter of the same name.  Ignored.
         opener: object, optional
             Mimicks built-in open parameter of the same name.  Ignored.
-        ignore_ext: boolean, optional
-            Disable transparent compression/decompression based on the file extension.
+        compression: str, optional (see smart_open.compression.get_supported_compression_types)
+            Explicitly specify the compression/decompression behavior.
         transport_params: dict, optional
             Additional parameters for the transport layer (see notes below).
-        
+
         Returns
         -------
         A file-like object.
-        
+
         Notes
         -----
         smart_open has several implementations for its transport layer (e.g. S3, HTTP).
         Each transport layer has a different set of keyword arguments for overriding
         default behavior.  If you specify a keyword argument that is *not* supported
         by the transport layer being used, smart_open will ignore that argument and
         log a warning message.
-        
+
         smart_open supports the following transport mechanisms:
-        
+
         azure (smart_open/azure.py)
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
         Implements file-like objects for reading and writing to/from Azure Blob Storage.
-        
+
+        blob_kwargs: dict, optional
+            Additional parameters to pass to `BlobClient.commit_block_list`.
+            For writing only.
         buffer_size: int, optional
             The buffer size to use when performing I/O. For reading only.
         min_part_size: int, optional
-            The minimum part size for multipart uploads.  For writing only.
-        
+            The minimum part size for multipart uploads. For writing only.
+        max_concurrency: int, optional
+            The number of parallel connections with which to download. For reading only.
+
         file (smart_open/local_file.py)
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
         Implements the transport for the file:// schema.
-        
+
+        ftp (smart_open/ftp.py)
+        ~~~~~~~~~~~~~~~~~~~~~~~
+        Implements I/O streams over FTP.
```

Comment:
@mpenkov do you know why `help` is picking up the [top-level ftp docstring](https://github.com/piskvorky/smart_open/blob/c299079da1f316ea86a3312a749b92c823b93c60/smart_open/ftp.py#L9), and not the kwargs of `ftp.open`? same for hdfs

Reply:
aha I see: https://github.com/piskvorky/smart_open/blob/c299079da1f316ea86a3312a749b92c823b93c60/smart_open/doctools.py#L47

Reply:
https://github.com/piskvorky/smart_open/pull/853/commits/6695cfd3afd6dc36f8ef2fc3261f8949ce1a0387